www/docs/server/api-handler.md

```diff
@@ -0,0 +1,67 @@
+---
+id: api-handler
+title: API Handler
+sidebar_label: API Handler
+slug: /api-handler
+---
+
+tRPC is not a backend of its own, but rather lives inside of other backends such as Next.js or Express. Despite that, most of tRPC's features and syntax are the same no matter which backend you are using. The API handler, also called [adapter](/docs/adapters), enables this by acting as the glue between HTTP requests to your backend and tRPC.
+
+The API Handler sits on a route in your server (usually `/api/trpc`, but this is just a convention) and processes all requests to that route and its subroutes. It receives a request from the server, uses the `createContext` function to generate [context](./context), and then sends the request and context to a [procedure](./procedure) in the router.
```

Comment:
Links here are broken according to build error on Vercel

Should be https://trpc.io/docs/procedures

Reply:
```suggestion
The API Handler sits on a route in your server (usually `/api/trpc`, but this is just a convention) and processes all requests to that route and its subroutes. It receives a request from the server, uses the `createContext` function to generate [context](./context), and then sends the request and context to a [procedure](./procedures) in the router.
```